index.js

```diff
@@ -139,6 +139,29 @@ const hasDevDependency = (dependency, packageJson) => {
   )
 }
 
+const runSValues = [new RegExp('run-s')]
+
+const hasRunS = (packageJson) => {
+  if (hasDevDependency('npm-run-all', packageJson)) {
+    const scripts = packageJson.scripts
+    const betterScripts = packageJson.betterScripts
+
+    if (scripts) {
+      return Object.values(scripts).some((script) =>
+        runSValues.some((runSValue) => runSValue.test(script)),
+      )
+    }
+
+    if (betterScripts) {
+      return Object.values(betterScripts).some((script) =>
+        runSValues.some((runSValue) => runSValue.test(script)),
+      )
+    }
+  }
+
+  return false
```

Comment:
What about an early return here?

```suggestion
  if (!hasDevDependency('npm-run-all', packageJson)) {
    return false
  }

  const scripts = packageJson.scripts
  const betterScripts = packageJson.betterScripts

  if (scripts) {
    return Object.values(scripts).some((script) =>
      runSValues.some((runSValue) => runSValue.test(script)),
    )
  }

  if (betterScripts) {
    return Object.values(betterScripts).some((script) =>
      runSValues.some((runSValue) => runSValue.test(script)),
    )
  }
```